martor/static/martor/js/martor.js

```diff
@@ -145,9 +127,9 @@
             }
             // Set autocomplete for ace editor
             if (editorConfig.mention === 'true') {
-                editor.completers = [emojiWordCompleter, mentionWordCompleter]
+                editor.completers = [mentionWordCompleter]
             }else {
```

Comment:
Let's autoformat this js at some point... Disgusting...

Reply:
#2388